### DISCUSSION TITLE
Has anyone noticed that output starts to get weird after swapping SD models a few times?

### DISCUSSION THREAD

**Discussion by keenerb** [Mikubill/sd-webui-controlnet#619] [General]
If I start A1111 ui, pick a model, and work with Controlnet and images I get good outputs for a long time.  Once I start switching between SD models (not controlnet models) outputs start having issues like neon colors, faces that are just a blur.

Maybe I'm doing something wrong, or not following the directions properly?

**Comment by CCpt5** [comment]
Interesting as I've been trying to pinpoint an issue w poor output on some occasions. If you switch models a bunch and go back to the first one does that one then produce lesser quality issues also?

Any chance you've resolved your problem?
